swap_meet/decor.py

```diff
@@ -1,2 +1,10 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+class Decor(Item):
+    def __init__(self, id=None, width=None, length=None, condition=None, age=None):
+        super().__init__(id, condition, age)
+        self.width = 0 if width is None else width
+        self.length = 0 if length is None else length
```

Comment:
Since integers are not mutable datatypes we don't need to take the same precautions as we do with mutable datatypes like lists. This is because a new `0` would be created in memory. So we could have the following:
```python
def __init__(self, id=None, width=0, length=0, condition=None, age=None):
        super().__init__(id, condition, age)
        self.width = width
        self.length = length
```